modules/ROOT/pages/deploying-to-multiple-environments.adoc

```diff
@@ -0,0 +1,234 @@
+
+You can test that your application uses the specific environment you configured in your variables properties, by running the application in Mule embedded in Studio.
+
+. In the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Mule Application*.
+
+. Studio automatically deploys your application according to the environment variable you specified in the *Global Property* from the previous procedure.
+
+. Again in the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Run Configurations...*.
++
+image::deploying-to-multiple-environments-07.png[]
+
+. Define the System Property in the `Arguments` tab.
++
+image::deploying-to-multiple-environments-08.png[]
+
+. Click the `Run` button to complete the procedure. The properties configured in `prod.yaml` will be used instead of the ones defined in `qa.yaml`.
+
+== Deploying Mule Apps using your Specific Environment Configurations
+
+Mule loads the environment variables when it starts, so depending on your deployment target (CloudHub or On-premises) you need to set the environment when you deploy the app (CloudHub) or when you start your Mule instance (On-premises).
+
+=== Setting a Specific Environment when Starting Mule On-Premises
+
+==== Using the command line
+
+Identify the environment in which to deploy your application at runtime with an environment variable. Execute the command to run Mule in your environment as per the following example:
+
+[source,console,linenums]
+----
+$ mule -M-Dmule.env=prod
+----
+
+Once Mule starts with your configured environment, you can deploy the application with your desired deployment method.
+
+==== Using Runtime Manager
+
+As a prerequisite you need to xref:runtime-manager::servers-create.adoc[create a server in Runtime Manager] and link it to Mule. Then you can xref:runtime-manager::servers-properties-create.adoc[create a custom server property].
+
+
+=== Setting a Specific Environment when Deploying to CloudHub
+
+==== Deploying to CloudHub from Studio
+
+From Studio you can initiate the process to set your properties environment variables and deploy your application to CloudHub :
+
+. In the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Anypoint Platform* -> *Deploy to Cloud*.
++
+image::deploying-to-multiple-environments-09.png[]
+
+. Complete the requested data, and click the *Properties* tab.
+
+. Add a new environment variable. Set the *Key* to *env* and its value to the environment you wish to deploy to (*prod* in this example).
++
+image::deploying-to-multiple-environments-10.png[]
+
+. Click *Deploy Application* to deploy.
+
+==== Deploying to CloudHub from Runtime Manager
+
+If you deploy your application to CloudHub via Runtime Manager, you can xref:configuring-properties#setting-properties-values-in-runtime-manager.adoc[set properties values] using the Runtime Manager Console.
```

Comment:
Runtime Manager > Anypoint Runtime Manager
"If you" > "To"
"using the Runtime Manager Console." > "using Runtime Manager console."